app/controllers/CommentController.php

```diff
@@ -2,23 +2,24 @@
 class CommentController
 {
     private $commentService;
+    private $commentRepository;
     private $cocktailService;
 
-    public function __construct(CommentService $commentService, CocktailService $cocktailService)
+    public function __construct(CommentService $commentService, CommentRepository $commentRepository, CocktailService $cocktailService)
```

Comment:
I was thinking that we should adjust how dependencies are handled here to better align with the MVC + Repository pattern we’re following. Specifically, we should only inject the CommentService into the controller, instead of both the service and the repository. 

The controller’s role is to handle HTTP requests and responses. Any business logic or database access should be managed by the service and repository layers.
By relying only on the CommentService, the controller stays focused and avoids handling logic directly tied to the database.

Reply:
You are absolutely right. I shall make some adjustments to this.